httpx/api.py

```diff
@@ -32,9 +32,9 @@ def request(
     cert: CertTypes = None,
     verify: VerifyTypes = True,
     stream: bool = False,
-    trust_env: bool = True,
+    trust_env: bool = None,
 ) -> Response:
-    with Client() as client:
+    with Client(trust_env=trust_env) as client:
```

Comment:
I'd suggest passing `trust_env` in the `request` call here, instead. That'd feel more obvious to me.